ic-helpers/src/factory/state.rs

```diff
@@ -23,11 +23,11 @@ macro_rules! impl_factory_state_management {
     ( $state:ident, $bytecode:expr ) => {
         impl $state {
             pub fn stable_save(&self) {
-                ::ic_cdk::storage::stable_save((self,)).unwrap();
+                ::ic_kit::ic::stable_store((self,)).unwrap();
             }
 
             pub fn stable_restore() {
-                let (mut loaded,): (Self,) = ::ic_cdk::storage::stable_restore().unwrap();
+                let (mut loaded,): (Self,) = ::ic_kit::ic::stable_restore().unwrap();
```

Comment:
Can we get rid of these unwraps. Either use expect if it can't fail, otherwise we should return a Result here

Reply:
probably we can delete he macro?